docs/resources/job.md

```diff
@@ -7,51 +7,9 @@ The `databricks_job` resource allows you to manage [Databricks Jobs](https://doc
 
 ## Example Usage
 
-```hcl
-data "databricks_current_user" "me" {}
-data "databricks_spark_version" "latest" {}
-data "databricks_node_type" "smallest" {
-  local_disk = true
-}
-
-resource "databricks_notebook" "this" {
-  path     = "${data.databricks_current_user.me.home}/Terraform"
-  language = "PYTHON"
-  content_base64 = base64encode(<<-EOT
-    # created from ${abspath(path.module)}
-    display(spark.range(10))
-    EOT
-  )
-}
-
-resource "databricks_job" "this" {
-  name = "Terraform Demo (${data.databricks_current_user.me.alphanumeric})"
-
-  new_cluster {
-    num_workers   = 1
-    spark_version = data.databricks_spark_version.latest.id
-    node_type_id  = data.databricks_node_type.smallest.id
-  }
-
-  notebook_task {
-    notebook_path = databricks_notebook.this.path
-  }
-}
-
-output "notebook_url" {
-  value = databricks_notebook.this.url
-}
-
-output "job_url" {
-  value = databricks_job.this.url
-}
-```
+-> **Note** In Terraform configuration, it is recommended to define tasks in alphabetical order of their `task_key` arguments, so that you get consistent and readable diff. Whenever tasks are added or removed, or `task_key` is renamed, you'll observe a change in the majority of tasks. It's related to the fact that the current version of the provider treats `task` blocks as an ordered list. Alternatively, `task` block could have been an unordered set, though end-users would see the entire block replaced upon a change in single property of the task.
 
-## Jobs with Multiple Tasks
-
--> **Note** In terraform configuration, it is recommended to define tasks in alphabetical order of their `task_key` arguments, so that you get consistent and readable diff. Whenever tasks are added or removed, or `task_key` is renamed, you'll observe a change in the majority of tasks. It's related to the fact that the current version of the provider treats `task` blocks as an ordered list. Alternatively, `task` block could have been an unordered set, though end-users would see the entire block replaced upon a change in single property of the task.
-
-It is possible to create [jobs with multiple tasks](https://docs.databricks.com/data-engineering/jobs/jobs-user-guide.html) using `task` blocks:
+It is possible to create [a Databricks job](https://docs.databricks.com/data-engineering/jobs/jobs-user-guide.html) using `task` blocks. Every `task` block can have almost all available arguments with the addition of `task_key` attribute and `depends_on` blocks to define cross-task dependencies.
```

Comment:
I think that it needs some reformulation - like: "It's possible to create a Databricks jobs consisting of multiple tasks. Single task is defined with the `task` block. ..."

Reply:
I've attempted a rewrite, but this could get complex